connector-sdk/core/src/main/java/io/camunda/connector/api/inbound/HealthError.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.api.inbound;
+
+public record HealthError(Severity severity, String text, String log, String title) {}
```

Comment:
Why should we separate between `text` and `log`? Do we need a `title`?

Reply:
My initial thought was having  a short version and longer version of information. In text we could add something like: NullPointerException. In the logs we would add something like the stacktrace. In this way, it would make sense to rename these to title, and stacktrace, but I would keep it as it is, since on the FE it would be beneficial to show the short version while having the possibility to show the longer version if requested. We might not need the title. @sbuettner Wdyt?

Reply:
@markfarkas-camunda I thought we only need one string and support expanding it on the client side.

Reply:
@sbuettner I thought we need at least 2 strings, since in the design I can see that there is a title written in bold, so we definitely have to differentiate between that and the longer text below it. If we only have 1 text, we have no chance to write anything in bold. So I would say we need 2 strings, rn I cannot see any other way. WDYT?

![image](https://github.com/camunda/connectors/assets/119574841/62d9882a-c433-4a45-b5ba-49bb6ae45ab7)

Reply:
Or is this title always just: webhook inactive? If so than I misunderstood and you are right, one text is enough 🙂

Reply:
I think we should keep it simple for the first iteration and only have a message

Reply:
As discussed in meeting with product, we keep a `code` property.